.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Report a bug encountered while running prometheus-adapter
+title: ''
+labels: kind/bug
+assignees: ''
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately see https://github.com/kubernetes/kube-state-metrics/blob/master/SECURITY.md
+-->
+
+**What happened?**:
+
+**What did you expect to happen?**:
+
+**Please provide the prometheus-adapter config**:
+<details open>
+<summary>prometheus-adapter config</summary>
```

Comment:
would you mind adding a comment here so that users know where to insert their manifests. A similar bug report template is used by metrics-server, maybe we could reuse some of their ideas: https://github.com/kubernetes-sigs/metrics-server/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

Reply:
Done :+1: